public/app/features/alerting/unified/api/k8s/timeinterval/v0alpha1/types.metadata.gen.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * metadata contains embedded CommonMetadata and can be extended with custom string fields
+ * TODO: use CommonMetadata instead of redefining here; currently needs to be defined here
+ * without external reference as using the CommonMetadata reference breaks thema codegen.
+ */
+export interface Metadata {
```

Comment:
Where do the names and comments come from exactly?

Reply:
generator [in app-sdk](https://github.com/grafana/grafana-app-sdk) https://github.com/grafana/grafana-app-sdk/blob/9303f0a9fb2f1fbd3242e0a4b93c6d24111f5558/codegen/cuekind/def.cue#L26